### PR TITLE
Updates to Luxon logic

### PR DESCRIPTION
**Pull request by kierangillen**
- Checks if hour is zero and returns 12
- Doesn't show 0 state times for countdown clock (e.g. `1d 0h 30m 20s` is now `1d 30m 20s`)
- Adds tests for `TimeRemaining`

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-palette* ready!

Built with commit 0d8749104b8e8dfa21be26af2f3d5bf7b6a65061

https://deploy-preview-502--artsy-palette.netlify.com

**Comment by artsyit** [comment]
Deploy preview for *artsy-palette* ready!

Built with commit 9befb63f35f38f21039992f33af12cc06fe739f2

https://deploy-preview-502--artsy-palette.netlify.com

File: packages/palette/src/elements/StaticCountdownTimer/StaticCountdownTimer.tsx [hunk 1]
```diff
@@ -24,7 +24,14 @@ export const StaticCountdownTimer: React.SFC<{
...
   const amPm = dateTime.hour >= 12 ? "pm" : "am"
-  const hour = dateTime.hour > 12 ? dateTime.hour - 12 : dateTime.hour
+  let hour
+  if (dateTime.hour > 12) {
+    hour = dateTime.hour - 12
+  } else if (dateTime.hour === 0) {
+    hour = 12
+  } else {
+    hour = dateTime.hour
+  }
```
Comment (damassi) [inline packages/palette/src/elements/StaticCountdownTimer/StaticCountdownTimer.tsx]: if / else is always better than a ternary 👍

**Comment by artsyit** [comment]
Deploy preview for *artsy-palette* ready!

Built with commit 76dbdb2ecf171fd6791983ec0dd9de3e58cbf2a7

https://deploy-preview-502--artsy-palette.netlify.com

File: packages/palette/src/elements/TimeRemaining/__tests__/TimeRemaining.test.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import { mount } from "enzyme"
+import { Settings } from "luxon"
+import React from "react"
+import { TimeRemaining } from "../TimeRemaining"
+
+describe("TimeRemaining", () => {
+  beforeEach(() => {
+    Settings.defaultZoneName = "America/New_York"
+  })
```
Comment (damassi) [inline packages/palette/src/elements/TimeRemaining/__tests__/TimeRemaining.test.tsx]:
```
let defaultZoneName

  before(() => {
    Settings.defaultZoneName = "America/New_York"
  })

  after(() => {
    Settings.defaultZoneName = defaultZoneName 
  })

...
```
Need to reset it back to how it was so other tests aren't polluted.

**Comment by artsyit** [comment]
:rocket: PR was released in v4.14.7 :rocket:

**Pull request merged**